libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/OBMySQLSchemaAccessor.java

```diff
@@ -179,6 +179,15 @@ public List<DBTableIndex> listTableIndexes(String schemaName, String tableName)
         return indexList;
     }
 
+    @Override
+    protected void handleIndexAvailability(DBTableIndex index, String availability) {
```

Comment:
duplicated code

Reply:
`OBMySQLSchemaAccessor#handleIndexAvailability`  override `MySQLNoGreaterThan5740SchemaAccessor#handleIndexAvailability`,  the implementation of these two methods is different.